### PR TITLE
Issue 134

### PR DESCRIPTION
**Pull request by lucasmichot**
WIP on #134 

---

All tests pass : https://travis-ci.org/danielstjules/Stringy/builds/124494966

---

Might be worth to SSH into the TravisCI container, if possible

### CODE REVIEW

**Comment by lucasmichot** [comment]
Ping @danielstjules

**Comment by GrahamCampbell** [comment]
mcrypt?

**Comment by GrahamCampbell** [comment]
mbstring?

File: .travis.yml [hunk 1]
```diff
@@ -39,5 +39,5 @@ before_script:
   - if [[ $polyfill = 'false' ]]; then travis_retry composer remove symfony/polyfill-mbstring; fi
 
 script:
-  - vendor/bin/phpunit
+  - vendor/bin/phpunit --verbose
```
Comment (GrahamCampbell) [inline .travis.yml]: better to set this in the phpunit config

File: .travis.yml [hunk 2]
```diff
@@ -36,8 +36,9 @@ install: travis_retry composer install --no-interaction --prefer-source
 before_script:
   - travis_retry composer self-update
   - travis_retry composer install --no-interaction --prefer-source
+  - if [[ $polyfill = 'true' ]]; then travis_disable_php_ext mcrypt; fi
```
Comment (danielstjules) [inline .travis.yml]: What is `travis_disable_php_ext mcrypt` ? The functionality from https://github.com/travis-ci/travis-ci/issues/4701 hasn't yet been implemented

Comment (GrahamCampbell) [inline .travis.yml]: you need to disable mbstring anyway, not mcrypt

**Comment by lucasmichot** [comment]
@GrahamCampbell - it's a WIP
Yeah I know this is mbstring
But I am mainly trying to find a way to do so, and mcrypt is by default on all Travis container

**Comment by danielstjules** [comment]
> and mcrypt is by default on all Travis container

As is mbstring. Anyway, I'm not worried about it for now, see https://github.com/danielstjules/Stringy/issues/134#issuecomment-212506366 My bigger concern is either updating the polyfill or getting rid of it.

**Comment by lucasmichot** [comment]
> My bigger concern is either updating the polyfill or getting rid of it.

At some point, I feel polyfill support should be removed

**Comment by danielstjules** [comment]
Closing for now - don't worry about it :) Travis doesn't support disabling extensions, and it's not worth compiling our own PHP from scratch.

**Pull request closed without merging**